### PR TITLE
Add ADS8688

### PR DESCRIPTION
**Pull request by matejx**
##### SUMMARY
Added TI ADS8688 symbol+component+device

##### CHECKLIST

- [x] I have read and followed the [library conventions](https://docs.librepcb.org/#libraryconventions).
- [ ] For packages, I followed IPC7351C (see details in library conventions).
- [x] I'm the copyright owner of the added content (i.e. the changes are made by myself, not copied/imported from somewhere else).
- [x] I agree to publish all my changes under the [CC0 Public Domain License](https://en.wikipedia.org/wiki/CC0), allowing everyone to use and modify the content without any restrictions.

### CODE REVIEW

File: sym/13e747a6-3ce1-4722-a18e-4fc469c491b0/symbol.lp [hunk 1]
```diff
@@ -0,0 +1,120 @@
+(librepcb_symbol 13e747a6-3ce1-4722-a18e-4fc469c491b0
+ (name "ADS8868")
```
Comment (dbrgn) [inline sym/13e747a6-3ce1-4722-a18e-4fc469c491b0/symbol.lp]: Typo, this should probably be 8688.

```suggestion
 (name "ADS8688")
```

File: sym/13e747a6-3ce1-4722-a18e-4fc469c491b0/symbol.lp [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+ (description "16-Bit 500kSPS 8-Channel Single-Supply SAR ADC With Bipolar Input Ranges")
+ (keywords "spi,adc,bipolar")
+ (author "Matej K.")
+ (version "0.1")
+ (created 2020-04-11T21:57:38Z)
+ (deprecated false)
+ (category a621d579-8715-4edf-816e-5715f97cd2d7)
+ (pin 809f7f16-6b76-4d9d-94bd-bfd3b673c67e (name "AIN0_P")
```
Comment (dbrgn) [inline sym/13e747a6-3ce1-4722-a18e-4fc469c491b0/symbol.lp:10]: More of a comment, not a change request: The datasheet inconsistently names the analog pins (e.g. `AIN_6P` but `AIN2_GND`). I guess `AINx_(P|GND)` makes most sense.

Edit: Woah, that datasheet is horrible. Sometimes they even mix up the pin names in the descriptions...

**Review by dbrgn — CHANGES_REQUESTED** [review]
Great, thanks for the contribution! Looks clean. Some notes:

- It seems that the datasheet recommends decoupling all AIN pins with a capacitor. Thus, I'd recommend doubling the spacing of those pins so that you can directly put a capacitor between a pair (the symbol has 5.08 spacing).
- For the same reason, space out REFIO/REFCAP/REFGND, AVDD/AGND, DVDD/DGND and AUX_IN/AUX_GND. The symbol will be less compact, but schematics will look better.

Regarding the pinout, do you have a reliable source? [That datasheet](http://www.ti.com/lit/ds/symlink/ads8688.pdf?ts=1588710326429) is the worst thing I've ever seen. Sometimes the description has pin names swapped. And while the pin functions table lists pin 3 as DAISY and pin 4 as nREFSEL, the layout example has them the other way around:

2020-05-05-224826_956x446_scrot

Apparently in the 5 years since that was published, either nobody complained or they just didn't bother to fix it.

**Comment by matejx** [comment]
I made the requested changes. I'm unsure how to proceed with the pull request, please advise.

The error in the datasheet was noticed by someone in 2018 and acknowledged by TI:

https://e2e.ti.com/support/data-converters/f/73/t/705308?ADS8688A-Possible-Pinout-Errata-in-Datasheet-Diagram

i.e. the suggested layout is wrong, the pin description is right.

**Review by dbrgn — APPROVED** [review]
Thanks for the errata link and the fixes! Looking great now, sorry for the long delay in reviewing.

**Pull request merged**